index.html

```diff
@@ -1498,6 +1515,60 @@
 
 <aside class="issue">Some services have the user pay for their use in data. These services aren't necessarily retaliating by denying their services to users who refuse to pay with data, but the details are more complex than we've had time to write.</aside>
 
+## Support Choosing Which Information to Present {#support-choosing-info}
+
+<div class="practice">
+  <span class="practicelab" id="principle-support-choosing-info">
+    [=User agents=] should support [=people=] in choosing which information they provide to [=parties=] that
+    request it, up to and including allowing for the provision of information that may not be factually
+    correct.
```

Comment:
```suggestion
    request it, up to and including allowing users to provide incorrect information.
```

Reply:
"arbitrary" (suggested by Pete, and liked by several)

Reply:
```suggestion
    request it, up to and including allowing users to provide arbitrary information.
```